frontends/web/src/routes/settings/settings.tsx

```diff
@@ -277,37 +277,30 @@ class Settings extends Component<Props, State> {
                           ) : null}
                           <h3 className="subTitle">{t('settings.expert.title')}</h3>
                           <div className="box slim divide m-bottom-large">
-                            <div className={style.setting}>
-                              <div>
-                                <p className="m-none">{t('settings.expert.fee')}</p>
-                              </div>
-                              <Toggle
-                                checked={config.frontend.expertFee}
-                                id="expertFee"
-                                onChange={this.handleToggleFrontendSetting} />
-                            </div>
-                            <div className={style.setting}>
+                            <SettingsToggle
+                              checked={config.frontend.expertFee}
+                              id="expertFee"
+                              onChange={this.handleToggleFrontendSetting}>
+                              {t('settings.expert.fee')}
+                            </SettingsToggle>
+                            <SettingsToggle
+                              checked={config.frontend.coinControl}
+                              id="coinControl"
+                              onChange={this.handleToggleFrontendSetting}>
                               <div>
                                 <p className="m-none">{t('settings.expert.coinControl')}</p>
                                 <p className="m-none">
                                   <Badge type="generic">BTC</Badge>
                                   <Badge type="generic" className="m-left-quarter">LTC</Badge>
                                 </p>
                               </div>
-                              <Toggle
-                                checked={config.frontend.coinControl}
-                                id="coinControl"
-                                onChange={this.handleToggleFrontendSetting} />
-                            </div>
-                            <div className={style.setting}>
-                              <div>
-                                <p className="m-none">{t('settings.expert.useSats')}</p>
-                              </div>
-                              <Toggle
-                                checked={config.backend.btcUnit === 'sat'}
-                                id="satsUnit"
-                                onChange={this.handleToggleSatsUnit} />
-                            </div>
+                            </SettingsToggle>
+                            <SettingsToggle
+                              checked={config.backend.btcUnit === 'sat'}
+                              id="satsUnit"
+                              onChange={this.handleToggleSatsUnit}>
+                              {t('settings.expert.useSats')}
```

Comment:
doens't use `<div><p className="m-none">` anymore (when not needed).

the `<p>` element was only used to apply the correct font styling, this is now done directly in the SettingsToggle.tsx component.